### DISCUSSION TITLE
How to use request.queue after current song and not in the middle?

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#2416] [General]
When I use the request.queue the new song play in the middle of the current, how to add after the current and after back to the playlist next song?

```
requests = request.queue(id="req_song")
radio = fallback(id="gerenciador_sources", track_sensitive = false, [requests, playlist])
```

**Comment by millicent-audio-mirror** [comment]
Hi @caf2022 , try changing track_sensitive to true in the fallback, making the move from playlist to request, and from request to playlist track sensitive i.e. it will wait until the end of the playing track until it plays from the next source.

The book is very good

https://www.liquidsoap.info/doc-dev/book.html

as are the docs

https://www.liquidsoap.info/doc-dev/reference.html#fallback

**Reply by caf2022** [reply]
Thanks for your help, but I have a question, using the track_sensitive as true, can affect other sources? Because in reality I'm using more sources:

`streaming = fallback(id="gerenciador_sources", track_sensitive = false, [dj, relay, playlist])`

If a DJ connect then will go to live, if I start a relay with input.http will go to relay, then if I change to true, the live and relay will start too only after current song ends?

**Reply by caf2022** [reply]
I tried that your sugested and not worked the request play only after the entire playlist ends.

I need to put the request on the next song in queue for example:
```
playlist.pls.next
[playing] /home/musics/4-non-blondes-whats-up.mp3
[ready] HERE
END
```
